concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1098,6 +1100,35 @@ jobs:
                 export CONFIG="$(pwd)/test-config/config.json"
                 ./paas-cf/tests/acceptance-tests/run_tests.sh
 
+  - name: bosh-cli-test
+    plan:
+      - aggregate:
+          - get: pipeline-trigger
+            passed: ['cf-deploy']
+            trigger: true
+          - get: paas-cf
+            passed: ['cf-deploy']
+          - get: concourse-manifest
+      - task: connect-to-bosh-cli
+        config:
+          platform: linux
+          image: docker:///governmentpaas/curl-ssl
+          params:
+            SYSTEM_DNS_ZONE_NAME: {{system_dns_zone_name}}
+          inputs:
+          - name: paas-cf
+          - name: concourse-manifest
+          run:
+            path: sh
+            args:
+            - -e
+            - -c
+            - |
+              apk update && apk add bash
+              export CONCOURSE_ATC_PASSWORD=$(awk '$1~/basic_auth_password/ {print $2}' concourse-manifest/concourse-manifest.yml)
```

Comment:
Can you make this consistent with https://github.com/alphagov/paas-cf/blob/d6e7de881a2664f04dbd4e38c79adb21dca5f3fc/concourse/scripts/self-update-pipeline.sh#L26-L27 ?